js/stanford_events_importer.js

```diff
@@ -231,37 +231,36 @@
 // -----------------------------------------------------------------------------
 
   if(typeof $.fn.quiet_highlight !== "function") {
-      /**
+    /**
      * A quick yellow highlight that fades out.
      */
     $.fn.quiet_highlight = function() {
```

Comment:
This is for the date/timepicker? If so, why is it in stanford_events_importer and not in stanford_date_timepicker?